samples/Actor/ActorClient/Program.cs

```diff
@@ -29,6 +29,9 @@ public static async Task Main(string[] args)
                 PropertyB = "ValueB",
             };
 
+            // Register custom serializers for the ActorProxy
+            //ActorProxy.JsonSerializerOptions.Converters.Add(new MyJsonConverter());
+
```

Comment:
Remove unused code

Reply:
Hey @vinayada1 - I  added those comments intentionally to act as guide for people that want to add JsonSerialization options. If I un-commented it I would need to implement a custom serializer so the code compiles. Maybe I can change the example to just set camel case to true?